src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/MethodTypeInference.cs

```diff
@@ -411,12 +411,12 @@ private bool ValidIndex(int index)
         private bool IsUnfixed(int methodTypeParameterIndex)
         {
             Debug.Assert(ValidIndex(methodTypeParameterIndex));
-            return _fixedResults[methodTypeParameterIndex].IsNull;
+            return _fixedResults[methodTypeParameterIndex].IsDefault;
         }
 
         private bool IsUnfixedTypeParameter(TypeSymbolWithAnnotations type)
         {
-            Debug.Assert(!type.IsNull);
+            Debug.Assert(!type.IsDefault);
```

Comment:
>IsDefault [](start = 31, length = 9)

HasType?

Reply:
Almost all uses of IsDefault will change to !HasType in the next iteration.

---
In reply to: [260056397](https://github.com/dotnet/roslyn/pull/33648#discussion_r260056397) [](ancestors = 260056397)